cqf-fhir-utility/src/test/java/org/opencds/cqf/fhir/utility/visitor/dstu3/ReleaseVisitorTests.java

```diff
@@ -296,68 +296,14 @@ void visitLibraryTest() {
-                "http://cts.nlm.nih.gov/fhir/ValueSet/2.16.840.1.113762.1.4.1146.1603|2022-10-19",
-                "http://cts.nlm.nih.gov/fhir/ValueSet/2.16.840.1.113762.1.4.1146.1602|2022-10-19",
-                "http://cts.nlm.nih.gov/fhir/ValueSet/2.16.840.1.113762.1.4.1146.1082|2022-10-19",
-                "http://cts.nlm.nih.gov/fhir/ValueSet/2.16.840.1.113762.1.4.1146.1439|2022-10-19",
-                "http://cts.nlm.nih.gov/fhir/ValueSet/2.16.840.1.113762.1.4.1146.1436|2022-10-19",
-                "http://cts.nlm.nih.gov/fhir/ValueSet/2.16.840.1.113762.1.4.1146.1435|2022-10-19",
-                "http://cts.nlm.nih.gov/fhir/ValueSet/2.16.840.1.113762.1.4.1146.1446|2022-10-19",
-                "http://cts.nlm.nih.gov/fhir/ValueSet/2.16.840.1.113762.1.4.1146.1438|2022-10-19",
-                "http://notOwnedTest.com/Library/notOwnedRoot|0.1.1",
-                "http://notOwnedTest.com/Library/notOwnedLeaf|0.1.1",
-                "http://notOwnedTest.com/Library/notOwnedLeaf1|0.1.1",
-                "http://hl7.org/fhir/us/core/StructureDefinition/us-core-patient",
-                "http://hl7.org/fhir/us/core/StructureDefinition/us-core-condition",
-                "http://hl7.org/fhir/us/core/StructureDefinition/us-core-encounter",
-                "http://hl7.org/fhir/us/core/StructureDefinition/us-core-medicationrequest",
-                "http://hl7.org/fhir/us/core/StructureDefinition/us-core-immunization",
-                "http://hl7.org/fhir/us/core/StructureDefinition/us-core-observation-lab",
-                "http://hl7.org/fhir/us/core/StructureDefinition/us-core-diagnosticreport-lab",
-                "http://hl7.org/fhir/us/ecr/StructureDefinition/eicr-document-bundle",
-                "http://hl7.org/fhir/StructureDefinition/ServiceRequest",
-                "http://hl7.org/fhir/us/ecr/StructureDefinition/ersd-valueset-library",
-                "http://hl7.org/fhir/us/ecr/StructureDefinition/us-ph-triggering-valueset-library",
-                "http://hl7.org/fhir/us/ecr/StructureDefinition/ersd-plandefinition",
-                "http://hl7.org/fhir/us/ecr/StructureDefinition/us-ph-plandefinition",
-                "http://hl7.org/fhir/us/ecr/StructureDefinition/ersd-valueset",
-                "http://hl7.org/fhir/us/ecr/StructureDefinition/us-ph-triggering-valueset",
-                "http://snomed.info/sct",
-                "http://hl7.org/fhir/sid/icd-10-cm",
-                "http://loinc.org",
-                "http://www.nlm.nih.gov/research/umls/rxnorm");
+                "http://ersd.aimsplatform.org/fhir/PlanDefinition/us-ecr-specification|" + existingVersion,
+                "http://ersd.aimsplatform.org/fhir/Library/rctc|" + existingVersion,
+                "http://ersd.aimsplatform.org/fhir/ValueSet/dxtc|" + existingVersion,
+                "http://cts.nlm.nih.gov/fhir/ValueSet/2.16.840.1.113762.1.4.1146.6|20210526",
+                "http://snomed.info/sct");
```

Comment:
since this is also a test of the dependency tracing, would it be possible to also add at least one profile dep (e.g. http://hl7.org/fhir/us/ecr/StructureDefinition/us-ph-triggering-valueset-library) and a component reference which does not have the `isOwned` extension?